### ISSUE TITLE
validator considers the wrong HTML as correct

### ISSUE THREAD

**Issue by nordost8** [ryanpcmcquen/unclosedTagFinder#5]
Hello,
your validator considers the following HTML as correct:
`<div class="container"><div id="content"></div>`
although it should consider it incorrect.

**Comment by ryanpcmcquen** [comment]
Good catch, I won't be able to get to this quickly but would merge a PR fixing it.
